include/crocoddyl/multibody/contacts/contact-3d.hpp

```diff
@@ -8,41 +8,75 @@
 
 #ifndef CROCODDYL_MULTIBODY_CONTACTS_CONTACT_3D_HPP_
 #define CROCODDYL_MULTIBODY_CONTACTS_CONTACT_3D_HPP_
-
+#include "crocoddyl/multibody/fwd.hpp"
 #include <pinocchio/spatial/motion.hpp>
 #include <pinocchio/multibody/data.hpp>
-#include "crocoddyl/multibody/contact-base.hpp"
+#include <pinocchio/algorithm/frames.hpp>
```

Comment:
Why do we need them?
I think these pinocchio headers are included in crocoddyl/frame and crocoddyl/contact-base